seq2science/rules/quantification.smk

```diff
@@ -268,70 +210,180 @@ elif config["quantifier"] == "kallistobus":
               opts="-p -t "$(wc -l {input.r1} | grep -Po '^\d+' | awk '{{print int($1/4)}}')
             fi
             fastq_pair $opts {output.intermediates1} >> {log} 2>&1
+            gzip -c {output.intermediates3[0]} > {output.reads[0]} 2> {log}
+            gzip -c {output.intermediates3[1]} > {output.reads[1]} 2>> {log}
         """
-                
-                
-    def get_kb_dir(wildcards):
-        if 'kite' in config.get('ref',""):
-            return directory(expand("{genome_dir}/{{assembly}}/index/kallistobus/kite/", **config))     
-        else:
-            return directory(expand("{genome_dir}/{{assembly}}/index/kallistobus/", **config))
         
-
-    def get_kb_reads(wildcards):
+    def get_final_reads(wildcards):
         if wildcards.sample in merged_treps:
             return expand(f"{{trimmed_dir}}/{wildcards.sample}_{{fqext}}_trimmed.{{fqsuffix}}.gz", **config)
         else:
             return rules.fastq_pair.output.reads
+                                            
+                                            
```

Comment:
two lines to separate functions and classes, single whitelines within :innocent: